### ISSUE TITLE
False positive commented-out code error

### ISSUE THREAD

**Issue by paulschreiber** [squizlabs/PHP_CodeSniffer#2470]
This file generates an unexpected error:
```php
<?php

class Foobar {

	public static function test() {
		// "Sponsored" and "More from..."
		echo '</div></div>';
	}
}
```

```
$ phpcs /tmp/test.php --standard=Squiz --sniffs=Squiz.PHP.CommentedOutCode

FILE: /private/tmp/test.php
---------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
---------------------------------------------------------------------------
 6 | WARNING | This comment is 60% valid code; is this commented out code?
---------------------------------------------------------------------------
```

**Comment by jrfnl** [comment]
For this sniff, PHPCS takes the text of a comment and lets PHP tokenize it as code to see how much of it comes back as "valid" code parts, then it makes some adjustments to prevent false positives and calculates the percentage.

Because of the quotes around the text strings, they are both tokenized as `T_CONSTANT_ENCAPSED_STRING` and the word `and` is a valid [logical operator](https://www.php.net/manual/en/language.operators.logical.php) in PHP, so it's only the 2 spaces being disregarded which makes this snippet 60%, not 100% valid code.
